data/dfndata/npc/animals.dfn

```diff
@@ -916,4 +927,32 @@ RACE=9
 CONTROLSLOTS=1
 NPCAI=12
 ORIGIN=ml
+}
+
+//crane - Base NPC with shared DFN tags
+[base_crane]
+{
+GET=base_animals
+NAME=#//crane
+ID=0x0fe
+ORIGIN=se
+}
+
+[crane]
+{
+GETSE=crane_se
```

Comment:
Add one of these for each expansion, so adding NPC with wrong expansion selected doesn't break the server. We can update where these point at when we have definitions for different eras:

GETUO=crane_se
GETT2A=crane_se
GETUOR=crane_se
GETTD=crane_se
GETLBR=crane_se
GETAOS=crane_se
GETSE=crane_se
GETML=crane_se
GETSA=crane_se
GETHS=crane_se
GETTOL=crane_se